src/featureParameters/Data_CountRdsWithDiffAmount.featureParameterInteger

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FeatureParameterInteger xmlns="http://soap.sforce.com/2006/04/metadata">
+    <dataflowDirection>SubscriberToLmo</dataflowDirection>
+    <masterLabel>Data: Count Of Recurring Donations With A Differing Opportunity Amount</masterLabel>
```

Comment:
@sam-knox I'm not sure how important these labels are in these feature parameters, but let me know if you want me to change any of them

Reply:
@seanpat09 and @sam-knox These labels are not visible to Customers at all. They're actually not visible to us either. They appear in the Cumulus Packaging Org as the description for the Feature Parameter.